packages/dart/noports_core/lib/src/srv/srv_impl.dart

```diff
@@ -100,9 +101,12 @@ class SrvImplExec implements Srv<Process> {
       rvArgs.add('--bind-local-port');
     }
     Map<String, String> environment = {};
-    if (rvdAuthString != null) {
-      rvArgs.addAll(['--rv-auth']);
-      environment['RV_AUTH'] = rvdAuthString!;
+
+    if (relayAuthenticator != null) {
+      rvArgs.addAll(relayAuthenticator!.rvArgs);
+      for (final String name in relayAuthenticator!.envMap.keys) {
+        environment[name] = relayAuthenticator!.envMap[name]!;
+      }
```

Comment:
add the srv command-line args, and place the required variables into the environment for the child srv process to pick up